adafruit_mcp2515/__init__.py

```diff
@@ -406,7 +406,8 @@ def unread_message_count(self):
         Returns:
             int: The unread message count
         """
-        self._read_from_rx_buffers()
+        if len(self._unread_message_queue) == 0:
```

Comment:
Please add a comment here about why we wait. Do you think it is worth checking the status registers and adding them to the unread message count?

Reply:
Looks good! I agree it is ok to skip the round trip.